Add Unleash feature flags example (#527)
vercel/examples: edge-middleware/feature-flag-unleash/README.md

---
name: Feature flags with Unleash
slug: feature-flag-unleash
description: Set up Unleash Proxy for use in Middleware and Server Side Rendering.
framework: Next.js
useCase: Edge Middleware
css: Tailwind
deployUrl: https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/edge-middleware/feature-flag-unleash&env=UNLEASH_RELAY_SECRET,UNLEASH_SERVER_API_URL,UNLEASH_SERVER_API_TOKEN&project-name=feature-flag-unleash&repository-name=feature-flag-unleash
demoUrl: https://feature-flag-unleash.vercel.app
---

# Feature Flags with Unleash

> Setting up [Unleash](https://www.getunleash.io/) Proxy on Vercel Edge middleware. It puts the load of getting feature flag definitions on Edge Network Cache, decreasing traffic to the Unleash instance. This example does not cover Static Generation, which is another way of scaling your application.

Reviewer: ```suggestion
Use [Unleash](https://www.getunleash.io/) with Vercel Edge Middleware. This example puts the load of getting feature flag definitions on the Edge Network Cache, decreasing traffic to the Unleash instance.
```
